search/search-client-elasticsearch/pom.xml

```diff
@@ -28,6 +28,16 @@
       <artifactId>elasticsearch-java</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>org.apache.httpcomponents</groupId>
+      <artifactId>httpclient</artifactId>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.httpcomponents.client5</groupId>
+      <artifactId>httpclient5</artifactId>
+    </dependency>
+
```

Comment:
🔧 I think those are not the correct dependencies for the exception class. This is also why the CI is failing.